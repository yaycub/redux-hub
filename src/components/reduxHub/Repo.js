import React from 'react';

const Repo = ({ url, name }) => (
  <a ></a>
);

Repo.propTypes = {

};

export default Repo;
