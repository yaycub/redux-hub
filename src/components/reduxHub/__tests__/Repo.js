import React from 'react';
import { shallow } from 'enzyme';
import Repo from '../Repo';

describe('Repo Component', () => {
  it('should render a repo', () => {
    const wrapper = shallow(
      <Repo 
        name='a repo' 
        url='repo.repo' />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
