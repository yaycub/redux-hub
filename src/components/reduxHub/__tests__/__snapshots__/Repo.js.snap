// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repo Component should render a repo 1`] = `
<a
  href="repo.repo"
  rel="noopener noreferrer"
  target="_blank"
>
  a repo
</a>
`;
